packages/bpk-component-modal/src/BpkModalV2/BpKModal.module.scss

```diff
@@ -76,9 +76,12 @@
 }
 
 .bpk-modal {
+  width: 100%;
+  max-width: none;
+  height: 100%;
+  max-height: 100%;
```

Comment:
Default behaviour on mobile should be full screen. Don't know how I could miss this for months 😶‍🌫️